### На соревновании по биатлону один из трех спортсменов стреляет и попадает в мишень. Вероятность попадания для первого спортсмена равна 0.9, для второго — 0.8, для третьего — 0.6. Найти вероятность того, что выстрел произведен: a). первым спортсменом б). вторым спортсменом в). третьим спортсменом.

Найти вероятность того, что выстрел произведен:

a) первым спортсменом

б) вторым спортсменом

в) третьим спортсменом.

Найдем вероятности этих событий по теореме Байеса
A - выстрел был произведен один и он был точным

P(B1) = P(B2) = P(B3) = 1/3

P(A) = P(B1) * P(A|B1) + P(B2) * P(A|B2) + P(B3) * P(A|B3) - полная вероятность

После нахождения полной вероятности определим, что это был определенный стрелок при помощи формулы: P(Bn) * P(A|Bn) / P(A)

In [1]:
P_b1 = 1/3 # вероятность выстрела первого стрелка
P_b2 = 1/3 # вероятность выстрела второго стрелка
P_b3 = 1/3 # вероятность выстрела третьего стрелка

A = P_b1 * 0.9 + P_b2 * 0.8 + P_b3 * 0.6
first = (P_b1 * 0.9) / A 
second = (P_b2 * 0.8) / A 
third = (P_b3 * 0.6) / A 

first, second, third

(0.391304347826087, 0.3478260869565218, 0.2608695652173913)

Ответ: Вероятность того, что выстрел произведен:

первым стрелком = 39.1%
вторым стрелком = 34.8%
третьим стрелком = 26%